src/NHotkey.Wpf/HotkeyManager.cs

```diff
@@ -94,7 +94,8 @@ private HotkeyManager()
             var parameters = new HwndSourceParameters("Hotkey sink")
                              {
                                  HwndSourceHook = HandleMessage,
-                                 ParentWindow = HwndMessage
+                                 ParentWindow = HwndMessage,
+                                 WindowStyle = 0x08000000 // WS_EX_NOACTIVATE
```

Comment:
Actually, are you sure this works? WS_EX_NOACTIVATE is an [extended window style](https://learn.microsoft.com/en-us/windows/win32/winmsg/extended-window-styles), not a window style. It should be assigned to [`ExtendedWindowStyle`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.interop.hwndsourceparameters.extendedwindowstyle?view=windowsdesktop-8.0#system-windows-interop-hwndsourceparameters-extendedwindowstyle). Assigning this value to `WindowStyle` actually means `WS_DISABLED`...

Reply:
It doesn't thing to break anything, but since I don't know how to repro the issue you're trying to solve, I can't check if it actually fixes it.

Reply:
I'm going to revert this change for now, until we're sure it actually does what it's supposed to.